docs/tutorials/articles/sales_dashboard/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Creating a Sales Dashboard
+category: fundamentals
+data-keywords: gui vizelement chart navbar table layout part menu state multi-page callback
+short-description: Understand basic knowledge of Taipy by creating a multi-page sales dashboard.
+order: 1.5
+img: sales_dashboard/images/thumbnail.png
+---
+
+!!! note "Supported Python versions"
+    Taipy requires **Python 3.9** or newer.
+
+This tutorial focuses on creating a simple sales dashboard application. You'll learn about visual elements,
+interaction, styling, and multi-page applications.
+
+![Final Application](images/final_app.png){width=90% .tp-image-border}
+
+This tutorial is also available in video format:
```

Comment:
I'd like a section "Why Taipy?" like in the docs to push people to do this tutorial

Reply:
Makes sense, I'm adding it